src/components/NewExperiment.vue

```diff
@@ -758,7 +771,142 @@ sample	reaction	measurement	uncertainity
         });
       });
     },
-    createExperiment() {}
+    createExperiment() {
+      this.conditionTempIdsMap = {};
+      this.sampleTempIdsMap = {};
+      this.isLoading = true;
+      axios
+        .post(`${settings.apis.warehouse}/experiments`, this.experiment)
+        .then((response: AxiosResponse) => {
+          return Promise.all(this.postConditions(response.data.id));
+        })
+        .then(() => {
+          return Promise.all(this.postSamples());
+        })
+        .then(() => {
+          return Promise.all(this.postItems("fluxomics"));
+        })
+        .then(() => {
+          return Promise.all(this.postItems("metabolomics"));
+        })
+        .then(() => {
+          return Promise.all(this.postItems("uptakeSecretion"));
+        })
+        .then(() => {
+          return Promise.all(this.postItems("molarYields"));
+        })
+        .then(() => {
+          return Promise.all(this.postItems("growth"));
+        })
+        .catch(error => {
+          this.$store.commit("setPostError", error);
+        })
+        .then(() => (this.isLoading = false));
```

Comment:
It would be easy to use a progress bar here which increments for finished request. I'll bring this up in the review